rotkehlchen/exchanges/coinbase.py

```diff
@@ -921,7 +927,7 @@ def _deserialize_asset_movement(self, raw_data: dict[str, Any]) -> list[AssetMov
             fee = Fee(ZERO)
             tx_type = raw_data['type']  # not sure if fiat
             event_type: Literal[HistoryEventType.DEPOSIT, HistoryEventType.WITHDRAWAL]
-            if tx_type in ('send', 'fiat_withdrawal'):
+            if tx_type in ('send', 'fiat_withdrawal', 'cardspend', 'clawback', 'incentives_shared_clawback', 'pro_deposit'):  # noqa: E501
```

Comment:
What are you doing here?

Since when is a carsspend a withdrawal?

How is a clawback a withdrawal?

And how is a deposit a withdrawal?